go/starter/main.go

```diff
@@ -0,0 +1,41 @@
+package main
+
+import (
+	"context"
+	"log"
+
+	"go.temporal.io/sdk/client"
+
```

Comment:
Not the biggest fan of import separation like this, but no big deal (I personally prefer only one blank line to separate stdlib vs non-stlib, everything else together)